data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -95,6 +95,57 @@
         return execute(DEFAULT_DATABASE_NAME, command);
     }
 
+    @Override
+    public KustoOperationResult executeStreamingIngestFromBlob(String databaseName, String tableName, String blobUrl, ClientRequestProperties properties,
```

Comment:
Why is it a separate implementation from executeStreamingIngest? they're nearly identical. Combine as much as you can to one function